libs/entitlement/src/main/java/org/elasticsearch/entitlement/initialization/EntitlementInitialization.java

```diff
@@ -50,6 +69,62 @@ public static void initialize(Instrumentation inst) throws Exception {
         inst.retransformClasses(System.class);
     }
 
+    private static PolicyManager createPolicyManager() throws IOException {
+        Map<String, Policy> pluginPolicies = createPluginPolicies();
+
+        // TODO: What should the name be?
+        // TODO(ES-10031): Decide what goes in the elasticsearch default policy and extend it
+        var elasticsearchPolicy = new Policy("elasticsearch", List.of());
```

Comment:
How about "server"? Using "elasticsearch" implies it's the entire application, which it's not.

Reply:
The only issue I have with server is that it also covers lucene and possibly other libs, but I guess it's still server's policy.

Reply:
Yep, it covers deps of server, just like a plugin policy covers the deps of the plugin.